vm/src/types/zoo.rs

```diff
@@ -81,6 +81,7 @@ pub struct TypeZoo {
     pub none_type: PyTypeRef,
     pub not_implemented_type: PyTypeRef,
     pub generic_alias_type: PyTypeRef,
+    pub vec_buffer_type: PyTypeRef,
```

Comment:
@youknowone can you help me to move it away?

Reply:
I attached a commit for this